dev/integration_tests/keyboard_hot_restart/lib/main.dart

```diff
@@ -11,8 +13,27 @@ import 'package:flutter/material.dart';
 // //dev/devicelab/lib/tasks/keyboard_hot_restart_test.dart
 const bool forceKeyboard = true;
 
+bool? keyboardVisible;
+
 void main() {
   runApp(const MyApp());
+  unawaited(printKeyboardState());
+}
+
+Future<void> printKeyboardState() async {
+  while (true) {
+    await Future<void>.delayed(const Duration(seconds: 1));
```

Comment:
Do you have any idea how long this typically has to wait?

Reply:
On my machine, this usually takes 1-2 for each of the checks: initial open keyboard check and the final closed keyboard check